packages/astro/src/default/components/MobileContentToggle.astro

```diff
@@ -0,0 +1,50 @@
+---
+
+---
```

Comment:
Is this needed even though it's empty?

Reply:
Oh you're right, I think it isn't! :open_mouth: